components/Layout/LeftMenu.tsx

```diff
@@ -123,7 +123,7 @@ const LeftMenu: React.FC<LeftMenuProps> = ({ activeItem, isMenuExpanded, setIsMe
           {items.map((item) => {
             const selected = item.href === activeItem
             return (
-              <Link key={item.title} href={item.href} passHref>
+              <Link key={item.title} href={item.href} as={`${item.href}`} passHref>
```

Comment:
how does it fix #571 ?

Reply:
https://github.com/forbole/official-website/issues/214#issue-815251146
I added the `as` attribute in regards to a previous issues with Nextjs routing! 
@calvinkei  Pplease see this issue and let me know if you think this is applicable to this

Reply:
this shouldn't be related. `/staking` is not a dynamic route and the error returned is not a 404 error. And I cannot reproduce this issue so I have no idea what's the cause